__tests__/app.test.js

```diff
@@ -196,4 +196,80 @@ describe("App", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("should return array of comment objects", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then((response) => {
+          const body = response.body;
+          expect(Array.isArray(body.comments)).toBe(true);
+          expect(body.comments.length).toBe(11);
+        });
+    });
+    test("should return array of comment objects with correct article id", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then((response) => {
+          const body = response.body;
+          body.comments.forEach((comment) => {
+            expect(comment.article_id).toBe(1);
+          });
+        });
+    });
+    test("should return comment object with correct properties", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then((response) => {
+          const body = response.body;
+          body.comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
```

Comment:
These three tests could all be within one expect statement. You could check the length then go straight into a `forEach` with the `toMatchObject` then instead of `article_id: expect.any(Number)` you could have `article_id : 1` as you know this should be the the same for each comment object in the array